phoenix-core/src/main/java/org/apache/phoenix/coprocessor/generated/PTableProtos.java

```diff
@@ -3725,35 +3725,35 @@ public Builder clearPGuidePosts() {
      */
```

Comment:
If I remember it correctly, this is an autogenerated file?

Reply:
Yes.